.git-blame-ignore-revs

```diff
@@ -1,7 +1,20 @@
 # This file is read by GitHub and `git blame --ignore-revs-file`.
 # See: https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
 
-# Revert "Change brace space in clang-format (#2627)"
```

Comment:
This commit message is wrong, at least I don't see it `tig`, the correct one is `Change brace space in clang-format (#2627)`, which you can see on the right side